src/test/java/api/support/fixtures/RequestsFixture.java

```diff
@@ -216,6 +217,22 @@ public Response attemptPlaceTitleLevelHoldShelfRequest(UUID instanceId,
       .withPickupServicePointId(servicePointsFixture.cd1().getId()));
   }
 
+  public Response attemptPlaceHoldOrRecallTLR(UUID instanceId,
+    IndividualResource requester, RequestType requestType) {
+
+    RequestBuilder builder = new RequestBuilder()
+      .hold()
+      .fulfilToHoldShelf()
+      .titleRequestLevel()
+      .withInstanceId(instanceId)
+      .withNoItemId()
+      .withNoHoldingsRecordId()
+      .withRequesterId(requester.getId())
+      .withPickupServicePointId(servicePointsFixture.cd1().getId());
+
+    return attemptPlace(RequestType.HOLD == requestType ? builder : builder.recall());
```

Comment:
Again, this method will create a Recall request when `requestType != HOLD`. So if you pass in `PAGE`, you will still get a Recall request. We can at least validate `requestType` before trying to build a request.

Reply:
what we should return if requestType == PAGE, null? So that, I method name is given like **_attemptPlaceHoldOrRecallTLR_**

Reply:
done